skia/recorder/include/video_extractor.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef VIDEOEXTRACTOR_HPP
+#define VIDEOEXTRACTOR_HPP
+
+#include "extractor.hpp"
+#include "recorder_arguments.hpp"
+#include "writer.hpp"
+
+class VideoExtractor : public virtual RiveFrameExtractor
```

Comment:
What's the need for virtual here? I don't see anything that requires a diamond hierarchy...